.doc_gen/metadata/cross_metadata.yaml

```diff
@@ -1,4 +1,32 @@
 # zexi 0.4.0
+cross_AuroraJob:
+  title: Create an application that stores and queries data by using Amazon Aurora
+  title_abbrev: Create an app that stores and queries Amazon Aurora data
+  synopsis: create an application that stores and queries data by using Amazon Aurora.
+  languages:
+    Java:
+      versions:
+        - sdk_version: 2
+          block_content: cross_AuroraApp_Java_block.xml
+  services:
+    rds:
+    translate:
+cross_LambdaTracker:
+  title: Create workflows that sends notifications over multiple channels
```

Comment:
should be workflows that send, not sends

Reply:
fixed